packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDTutorial/PhotoCaptureHUDTutorial.tsx

```diff
@@ -0,0 +1,129 @@
+import { styles } from './PhotoCaptureHUDTutorial.styles';
+import { TutorialViews } from '../../hooks';
+import { usePhotoCaptureHUDButtonBackground } from '../hooks';
+import { SightGuideline } from '../PhotoCaptureHUDElementsSight';
+import { arrowGuidelineSVG, arrowSightTutorialSVG } from '../../../assets';
+
+const TRANSLATION_PREFIX = 'photo.hud.tutorial';
+
+/**
+ * Props of the PhotoCaptureHUDTutorial component.
+ */
+export interface PhotoCaptureHUDTutorialProps
+  extends Pick<
+    CaptureAppConfig,
+    'enableSightGuidelines' | 'allowSkipTutorial' | 'sightGuidelines' | 'enableSightTutorial'
+  > {
+  /**
+   * The id of the sight.
+   */
+  sightId: string;
+  /**
+   * The current tutorial view in PhotoCapture component.
+   */
+  tutorialView: TutorialViews | null;
+  /**
+   * Callback called when the user clicks on "Next" button.
+   */
+  onTutorialNext: (view: TutorialViews | null) => void;
+}
+
+function getButtonStyle(enableAddDamage?: boolean): CSSProperties {
+  return { visibility: enableAddDamage ? 'visible' : 'hidden' };
+}
+
+function ArrowIcon({ tutorialView }: Pick<PhotoCaptureHUDTutorialProps, 'tutorialView'>) {
+  if (tutorialView === TutorialViews.GUIDELINE) {
+    return <DynamicSVG style={styles['arrowGuideline']} svg={arrowGuidelineSVG} />;
+  }
+  if (tutorialView === TutorialViews.SIGHT_TUTORIAL) {
+    return <DynamicSVG style={styles['arrowSightTutorial']} svg={arrowSightTutorialSVG} />;
+  }
+  return null;
+}
+
+function DisplayText({ tutorialView }: Pick<PhotoCaptureHUDTutorialProps, 'tutorialView'>) {
+  const { t } = useTranslation();
+
+  const textArray = t(`${TRANSLATION_PREFIX}.${tutorialView}`).split(/<br\s*\/?>/);
+
+  return (
+    <div style={styles['text']}>
+      <div style={styles['title']} data-testid='title'>
+        {t(`${TRANSLATION_PREFIX}.title`)}
+      </div>
+      {textArray.map((value: string, index: number) => (
+        <div key={index}>
+          {value}
+          {index < textArray.length - 1 && <br />}
+        </div>
+      ))}
+    </div>
+  );
+}
+
+/**
+ * Component that displays an tutorial overlay on top of the PhotoCapture component.
+ */
+export function PhotoCaptureHUDTutorial({
+  tutorialView,
+  onTutorialNext,
+  allowSkipTutorial,
+  enableSightGuidelines,
+  enableSightTutorial,
+  sightGuidelines,
+  sightId,
+}: PhotoCaptureHUDTutorialProps) {
+  const { t } = useTranslation();
+  const primaryColor = usePhotoCaptureHUDButtonBackground();
+
+  const handleNext = () => {
+    let views = Object.values(TutorialViews);
+    if (!enableSightGuidelines) {
+      views = views.filter((v) => v !== TutorialViews.GUIDELINE);
+    }
+    if (!enableSightTutorial) {
+      views = views.filter((v) => v !== TutorialViews.SIGHT_TUTORIAL);
+    }
+    if (tutorialView === views.at(-1)) {
+      onTutorialNext(null);
+      return;
+    }
+    const currentIndex = views.findIndex((v) => tutorialView === v);
+    onTutorialNext(views[currentIndex + 1]);
+  };
```

Comment:
I think it would be cleaner if this logic was implemented in the `usePhotoCaptureTutorial` hook. Like all the logic to decide which tutorial step must be displayed, what is the next step etc, all of this should be in the hook.

I think instead of returning the `handleTutorialView` callback, the hook should return simple callback called `go ToNextTutorialStep` that accepts no parameter and that automatically handles the navigation between steps. To close the tutorial you can also return a callback called `closeTutorial` or something like that if you want